### DISCUSSION TITLE
zotero导出的md文件在obsidian中打开，点击文件中的链接无法跳转到正确的pdf位置

### DISCUSSION THREAD

**Discussion by liuyueChang** [windingwind/zotero-better-notes#1070] [Q&A]
zotero导出的md文件在obsidian中打开，点击文件中的链接无法跳转到正确的pdf位置

**Comment by windingwind** [comment]
请负责任地提问而不是抱怨，请详细描述

**Reply by windingwind** [reply]
任何认为是bug的问题，请提issue，谢谢。此处关闭

**Reply by liuyueChang** [reply]
better notes 导出md文件，导出配置如下图所示
![image](https://github.com/user-attachments/assets/ab881750-1b6b-479e-a5cf-cdb2c0bf435e)
该md文件中包含有pdf部分段落的链接，在obsidian中打开该md文件
点击该链接后，能够打开zotero，但是并不能跳转到正确的位置

环境如下：
windows11
zotero 7.0.0-beta.108+80cc97125
better notes 1.1.4-beta.109

如果需要其他信息请指出

**Reply by windingwind** [reply]
如上所示，请提issue，issue模板说明了所需要的信息。勿回复
